OpenSkyPlus.csproj

```diff
@@ -1,54 +1,81 @@
-  </ItemGroup>
-  
-  <ItemGroup Condition="'$(TargetFramework.TrimEnd(`0123456789`))' == 'net'">
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="all" />
-  </ItemGroup>
-  
-  <ItemGroup>
-    <Reference Include="UnityEngine.CoreModule">
-    </Reference>
-    <Reference Include="UnityEngine.UI">
-      <HintPath>lib\UnityEngine.UI.dll</HintPath>
-    </Reference>
-  </ItemGroup>
-  
-  <Import Project="AbstractOpenSkyPlusApi\AbstractOpenSkyPlusApi.projitems" Label="Shared" />
-  
-  <Import Project="OpenSkyPlusApi\OpenSkyPlusApi.projitems" Label="Shared" />
-  
-  <Import Project="OpenSkyPlusConfiguration\OpenSkyPlusConfiguration.projitems" Label="Shared" />
-  
-
-</Project>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	<GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+	<GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+	<TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>OpenSkyPlus</AssemblyName>
+    <Description>OpenSkyPlus for [LaunchMonitor]</Description>
+    <Version>1.0.1</Version>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="OpenSkyPlusAPI\**" />
+    <Compile Remove="OpenSkyPlusConfiguration\**" />
+    <Compile Remove="OpenSkyPlusInjector\**" />
+    <EmbeddedResource Remove="OpenSkyPlusAPI\**" />
+    <EmbeddedResource Remove="OpenSkyPlusConfiguration\**" />
+    <EmbeddedResource Remove="OpenSkyPlusInjector\**" />
+    <None Remove="OpenSkyPlusAPI\**" />
+    <None Remove="OpenSkyPlusConfiguration\**" />
+    <None Remove="OpenSkyPlusInjector\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="logo.png" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BepInEx.Analyzers" Version="1.*" PrivateAssets="all" />
+    <PackageReference Include="BepInEx.Core" Version="5.*" />
+    <PackageReference Include="BepInEx.PluginInfoProps" Version="1.*" />
+    <PackageReference Include="System.Runtime.Serialization.Json" Version="4.3.0" />
+    <PackageReference Include="System.Text.Json" Version="8.0.1" />
+    <PackageReference Include="UnityEngine.Modules" Version="2020.3.8" IncludeAssets="compile" />
+  </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework.TrimEnd(`0123456789`))' == 'net'">
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="all" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <Reference Include="UnityEngine.CoreModule">
+    </Reference>
+    <Reference Include="UnityEngine.UI">
+      <HintPath>lib\UnityEngine.UI.dll</HintPath>
+    </Reference>
+	  <Reference Include="Unity.TextMeshPro"/>
+  </ItemGroup>
+  
+  <ItemGroup>
+    <EmbeddedResource Include="logo.png">
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+    </EmbeddedResource>
+  </ItemGroup>
+  
+  <ItemGroup>
```

Comment:
This can all get removed if not using the resx file

Reply:
I embedded your bitmap in there, weird that it does not show up.  I will check if I have any pending changes

Reply:
Yeah, it should be in there then